pubspec.yaml

```
@@ -3,6 +3,8 @@ description: A package which simplifies the task of updating your Flutter app's
 version: 0.10.0
 maintainer: Mark O'Sullivan (@MarkOSullivan94)
 homepage: https://github.com/fluttercommunity/flutter_launcher_icons
+repository: https://github.com/fluttercommunity/flutter_launcher_icons/
+issue_tracker: https://github.com/fluttercommunity/flutter_launcher_icons/issues/new?assignees=&labels=&template=bug_report.md&title=%5BBUG%5D+
```

```suggestion
issue_tracker: https://github.com/fluttercommunity/flutter_launcher_icons/issues
```


[The optional issue_tracker field should contain a URL for the package’s issue tracker, where existing bugs can be viewed and new bugs can be filed.](https://dart.dev/tools/pub/pubspec#issue-tracker)